{"name": "Сообщение об ошибке при получении заказа по несуществующему трек-номеру", "status": "passed", "start": 1730968899538, "stop": 1730968899683, "uuid": "3c767a51-d5e1-4801-8a5d-86a98e5926b2", "historyId": "61e494ad045ec34d8f88ce7df6d40773", "testCaseId": "61e494ad045ec34d8f88ce7df6d40773", "fullName": "tests.test_orders.TestGetOrderByTrack#test_error_get_order_by_incorrect_track", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "MacBook-Air-Maria.local"}, {"name": "thread", "value": "28387-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders"}]}